### DISCUSSION TITLE
How to clear cache on react native?

### DISCUSSION THREAD

**Discussion by alimertcakar** [vercel/swr#1099] [Q&A]
`
import { cache } from "swr" ;
cache.clear();
`

I run this on logout, yet I still get the cached old user data. What am I doing wrong?

**Comment by rogueturnip** [comment]
Anyone have any suggestions on this?  Even just invalidating the cache so it shows nothing would be better then old data.

**Reply by shuding** [reply]
https://swr.vercel.app/docs/advanced/cache#access-to-the-cache

You can use `useSWRConfig` and `cache.clear`.
